docs/howto/manage-web-app-charms/use-web-app-charm.rst

```diff
@@ -28,3 +28,65 @@ script will run on every unit. The script is assumed to be idempotent (in other
 can be run multiple times) and that it can be run on multiple units simultaneously
 without issue. Handling multiple migration scripts that run concurrently
 can be achieved by, for example, locking any tables during the migration.
+
+Use observability
+-----------------
+
+First, :ref:`integrate your web app with the Canonical Observability
+Stack <integrate_web_app_cos>`.
+
+Get Grafana login details
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+View the endpoints by running:
+
+.. code-block::
+
+    juju show-unit catalogue/0 | grep url
+    juju run grafana/leader get-admin-password
```

Comment:
What do you think about splitting the steps between these two lines? To me these feel like separate actions.